src/BeastImage.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Image from 'react-bootstrap/Image';
+
+class Beast extends React.Component {
```

Comment:
I'm confused about your component Beast vs HornedBeast. Is HornedBeast just a depreciated component from your last lab? I'm fairly certain you should either delete or update one these.